bin/deploy.sh

```diff
@@ -84,7 +84,11 @@ fi
 ## Prod and dev sites: install the application.
 composer install --no-dev --optimize-autoloader
 ./bin/console cache:clear
-./bin/console doctrine:migrations:migrate --no-interaction
+
+## Run migration command if migration folder exists.
+if [[ -d "migrations" ]]; then
```

Comment:
This is the standard directory in newer versions of Symfony, but it can be configured to live anywhere. Provided migrations do exist and they're configured properly, it's safe to run `./bin/console doctrine:migrations:migrate --no-interaction`.

Maybe we should instead do like the we do for the assets and look for a flag like `--migrations`?

Reply:
That's a good point. I do wonder if we want to encourage multiple locations — there's a lot to be said for consistency I think, and probably all new tools will have it at `migrations/`.

Maybe rather than a new flag, this could just check for the two existing directories for migrations? e.g.

```bash
if [[ -d "migrations" || -d "app/DoctrineMigrations" ]]; then
```

What do you think?

Reply:
I'm fine with that! But it's worth noting the location at `migrations/` isn't a rule. Many apps built in older versions of Symfony may bypass path changes like this when they upgrade to newer versions of Symfony, solely because it saves time and is by no means necessary. In fact, Symfony acknowledges there may even be [_multiple_ migration paths](https://symfony.com/bundles/DoctrineMigrationsBundle/current/index.html#configuration) for a single app. For us (Community Tech) it makes total sense to keep it consistent, but the same can't be said for any Symfony app that may wish to use this bundle.

That said, I've no issue with just merging this as-is! Event Metrics is unlikely to have any new migrations, anyway.

Reply:
Okay sure, well let's go with this for now, and revisit it as required. It'll at least stop the errors we get on every new deploy of the OCR tool! :)